data/nlu.md

```diff
@@ -81,6 +80,145 @@ data/lookup_tables/provinces.txt
 - [western cape]{"entity": "province", "value": "wc"}
 - [1](number)
 - [2](number).
+- [2](number)
+- [3](number)
+- [9](number)
+- [cape](province) town
```

Comment:
@rudigiesler I think these 4 lines were added by the interactive learning, but I'm not sure. Please check

Reply:
Yes it was. `[cape](province) town` is also incorrect, so I'll remove these lines